### DISCUSSION TITLE
MPL_Lightning

### DISCUSSION THREAD

**Discussion by Harunercul** [community/community#103060] [Programming Help]
### Select Topic Area

General

### Body

Hi Dear ı have this problem . Please help me this error  = "python main.py --seed 5 --name cifar10-4K.5 --expand-labels --dataset cifar10 --num-labeled 4000 --total-steps 300000 --eval-step 1000 --randaug 2 16 --batch-size 128 --teacher_lr 0.05 --student_lr 0.05 --weight-decay 5e-4 --ema 0.995 --enable-nesterov --mu 7 --label-smoothing 0.15 --temperature 0.7 --threshold 0.6 --lambda-u 8  --uda-steps 5000  --teacher-dropout 0.2 --student-dropout 0.2 --workers 16
Seed set to 5
Traceback (most recent call last):
  File "C:\Users\HPWIN10\Desktop\mlp\main.py", line 98, in <module>
    model = get_model(args)
  File "C:\Users\HPWIN10\Desktop\mlp\main.py", line 68, in get_model
    mpl_model = mpl_lightning.LightningMPL(
  File "C:\Users\HPWIN10\Desktop\mlp\mpl_lightning\pl_modules.py", line 124, in __init__
    "train/top1_acc": metrics.Accuracy(top_k=1),
**TypeError: __new__() missing 1 required positional argument: 'task'"**      I am waiting your feedback.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Harunercul ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ghost** [comment]
It seems like there's an issue with the initialization of your LightningMPL model. The error indicates that the 'task' argument is missing. Ensure that you are passing all the required arguments correctly when creating the LightningMPL instance in your `pl_modules.py` file.

**Comment by Harunercul** [comment]
Okay ı am trying thank you again

2 Şub 2024 Cum, saat 20:42 tarihinde Yuhesh Pandian P J <
***@***.***> şunu yazdı:

**Reply by ghost** [reply]
Your welcome and All the Best in that
